### PR TITLE
Nested arrays javascript

### PR DESCRIPTION
**Pull request by JohnForster**
Provides functionality for nested arrays and chaining two methods.

### CODE REVIEW

**Review by bibicollins — APPROVED** [review]
:ant:

File: src/model.js [hunk 1]
```diff
@@ -21,32 +22,39 @@ function workOutType(string) {
...
+  return JSON.stringify(method.call(array1)) === JSON.stringify(array2);
 }
 
 function findMethod(inputArray, desiredOutput) {
   const outputArray = [];
-  methodList.forEach((method) => {
-    if (compareArrays(inputArray.slice(), desiredOutput, method)) {
-      outputArray.push(`.${method.name}`);
+  methodList.forEach((firstMethod) => {
+    if (compareArrays(inputArray.slice(), desiredOutput, firstMethod)) {
```
Comment (elishkaflint) [inline src/model.js]: careful! slice only makes a shallow copy, can you change? :)

**Review by elishkaflint — CHANGES_REQUESTED** [review]

File: src/model.js (hunk 1 above)
Comment (JohnForster) [reply src/model.js]: Ah! Good spot! Changed in the most recent commit.

**Review by elishkaflint — APPROVED** [review]
🐜

**Pull request merged**